micro-ui/web/micro-ui-internals/packages/modules/sandbox/src/configs/UICustomizations.js

```
@@ -0,0 +1,272 @@
+                to={`/${window.contextPath}/employee/workbench/mdms-search-v2?moduleName=${value?.split(".")?.[0]}&masterName=${
+                  value?.split(".")?.[1]
+                }`}
+              >
+                {t(Digit.Utils.workbench.getMDMSLabel(`SCHEMA_` + value))}
+                {/* {t(`SANDBOX_${value?.split(".")?.[0]}`)} */}
+              </Link>
+            </span>
+          );
+        case "SANDBOX_MASTER_NAME":
+          // return row?.code?.split(".")?.[1] ? t(`SANDBOX_${row?.code?.split(".")?.[1]}`) : t(`SANDBOX_${row?.code?.split(".")?.[0]}`);
+          return t(Digit.Utils.workbench.getMDMSLabel(`SCHEMA_` + row?.code));
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)